packages/integration/api-report.md

```diff
@@ -187,8 +191,15 @@ export type BitbucketCloudIntegrationConfig = {
   username?: string;
   appPassword?: string;
   token?: string;
+  fetch?: FetchConfig;
```

Comment:
if you could change this to only be `throttling?: ThrottlingConfig` or so, then that's a small enough change to let through.